lib/validators.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const Joi = require('joi');
+
+const errors = require('./errors');
+
+const validators = {schemas: {}};
+
+validators.schemas.requestData = Joi.object().keys({
```

Comment:
why is this called `requestData`?

Reply:
mm it's the request object that we're validating. So I guess a better name would be requestObject or requestBody? Lmk what you think is best. I'm leaning towards requestBody.

Reply:
I would say `AuthClient`? or `authConstructorArgs` because at a high level we're just validating the arguments to that function 

although if we move this into the method, this would be a moot point